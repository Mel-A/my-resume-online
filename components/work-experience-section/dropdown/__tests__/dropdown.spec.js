import { shallow } from 'enzyme'
import Dropdown from '..'

describe('dropdown', () => {
it('dropdown renders correctly', () => {
    const dropdown = shallow(<Dropdown />)
    expect(dropdown).toMatchSnapshot()
  })
})
