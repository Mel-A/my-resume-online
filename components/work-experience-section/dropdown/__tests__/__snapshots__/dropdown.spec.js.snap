// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dropdown dropdown renders correctly 1`] = `ShallowWrapper {}`;
